config/jobs/kubernetes-sigs/kjob/kjob-periodics-main.yaml

```diff
@@ -117,8 +117,8 @@ periodics:
             privileged: true
           resources:
             requests:
-              cpu: "7"
-              memory: "10Gi"
+              cpu: "2"
```

Comment:
Can you please justify the numbers in the PR description?

Looking at the monitoring, for example here: https://monitoring-eks.prow.k8s.io/d/96Q8oOOZk/builds?orgId=1&from=1742308764970&to=1742309265767&var-org=kubernetes-sigs&var-repo=kjob&var-job=periodic-kjob-test-e2e-main&var-build=All

![image](https://github.com/user-attachments/assets/10f3eea1-4590-40c0-9d05-efa90fa04cd3)

the e2e tests seem t use at peaks the 7cpu, so it will be running slower. Also, it is preferred to keep cpu and mem 1:1. Maybe 4cpu, and 4Gi we could start with, and see what is the impact?